<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Khối</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="khoi_ipt">
        <item>
         <property name="text">
          <string>6</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Lớp</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lop_ipt"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>GVCN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gvcn_ipt">
        <item>
         <property name="text">
          <string>Không có giáo viên trống</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>GV Văn</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gvV_ipt">
        <item>
         <property name="text">
          <string>Không có giáo viên trống</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>GV Toán</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gvT_ipt">
        <item>
         <property name="text">
          <string>Không có giáo viên trống</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>GV Anh</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gvA_ipt">
        <item>
         <property name="text">
          <string>Không có giáo viên trống</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>GV KHTN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gvK_ipt">
        <item>
         <property name="text">
          <string>Không có giáo viên trống</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
